Source/HYPFormsCollectionViewDataSource.m

```diff
@@ -78,11 +79,7 @@ - (instancetype)initWithCollectionView:(UICollectionView *)collectionView
 
 - (NSMutableArray *)collapsedForms
 {
-    if (_collapsedForms) return _collapsedForms;
-
-    _collapsedForms = [NSMutableArray new];
-
-    return _collapsedForms;
+    return _collapsedForms = (_collapsedForms) ?: [NSMutableArray new];
```

Comment:
```_collapsedForms``` either gets assigned the value of the conditional, it will fallback to creating a new mutable array.

 I think this is pretty clearcut :)